src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/RegistrationController.php

```diff
@@ -178,9 +179,25 @@ public function registrationPdfAction($secondFactorId)
                 'tempDir' => sys_get_temp_dir(),
             )
         );
+        $mpdf->setLogger($this->get('logger'));
```

Comment:
If we can use the solution you propose than please remove the suppression that was previously added.

>  @SuppressWarnings(PHPMD.ExitExpression) MPDF requires bypassing Symfony, so we exit() when MPDF is done.

See #134 for the original development effort.

I did not see any requests for additional logging from mpdf. Personally I think it's a good idea, did you run this past @pmeulen or another product owner?

Reply:
Yes, this issue wasn't logged either.